src/coreclr/jit/instr.cpp

```diff
@@ -103,7 +103,7 @@ const char* CodeGen::genInsDisplayName(emitter::instrDesc* id)
 
     const emitter* emit = GetEmitter();
 
-    if (emit->IsVexOrEvexEncodedInstruction(ins))
```

Comment:
`Encodable` is a bit more accurate (then `Encoded`) since we're doing a "can it be encoded" rather than a "must it be encoded" check.